### ISSUE TITLE
Fetch failed for 'www-client/yandex-browser-beta-17.7.1.716'

### ISSUE THREAD

**Issue by usern1983** [rasdark/overlay#3]
cat /var/calculate/tmp/portage/www-client/yandex-browser-beta-17.7.1.716/temp/build.log 
```
>>> Downloading 'http://repo.yandex.ru/yandex-browser/deb/pool/main/y/yandex-browser-beta/yandex-browser-beta_17.7.1.716-1_amd64.deb'
--2017-08-24 15:28:25--  http://repo.yandex.ru/yandex-browser/deb/pool/main/y/yandex-browser-beta/yandex-browser-beta_17.7.1.716-1_amd64.deb
Resolving repo.yandex.ru (repo.yandex.ru)... 2a02:6b8::183, 213.180.204.183
Connecting to repo.yandex.ru (repo.yandex.ru)|2a02:6b8::183|:80... connected.
HTTP request sent, awaiting response... 404 Not Found
2017-08-24 15:28:25 ERROR 404: Not Found.

!!! Couldn't download 'yandex-browser-beta-17.7.1.716.deb'. Aborting.
 * Fetch failed for 'www-client/yandex-browser-beta-17.7.1.716', Log file:
 *  '/var/calculate/tmp/portage/www-client/yandex-browser-beta-17.7.1.716/temp/build.log'

```

**Comment by rasdark** [comment]
yandex has change 17.7.1.716 with 17.6.1.835. I'll wait for the next release

**Comment by rasdark** [comment]
Fixed in rasdark@340befc96ce4339b95756da8feb45b178800dc8e
